crypto/fipsmodule/rand/new_rand_test.cc

```diff
@@ -9,12 +9,39 @@
 #include "new_rand_internal.h"
 #include "../../ube/internal.h"
 
+#include "../../test/test_util.h"
+
 #include <thread>
 
 // TODO
 // Remove when promoting to default
 #if !defined(BORINGSSL_PREFIX)
 
+class newRandTest : public ::testing::Test {
+  public:
+    void SetUp() override {
+      uint64_t current_generation_number = 0;
+      if (CRYPTO_get_ube_generation_number(&current_generation_number) == 1) {
+        ube_detection_supported_ = true;
+      }
+    }
+
+    void TearDown() override {
+      disable_mocked_ube_detection_FOR_TESTING();
+    }
```

Comment:
By default we run most of our tests in parallel, will modifying global state here cause an issue?

Reply:
We should be running process-isolated parallelism which means no issue.